### PR TITLE
Fixes broken links in SDK docs v3.0.2 - Closes #1282

### LINKED ISSUE #1282: Fixes broken links in SDK docs v3.0.2
Fixes broken links after updates in the SDK v3,02 documentation in the pages listed below:

lisk.com/documentation/lisk-sdk/v3/references/lisk-elements/index
lisk.com/documentation/lisk-sdk/v3/references/index

### PR DESCRIPTION
**Pull request by chris529**
Closes #1282

### CODE REVIEW

**Review by ishantiw — DISMISSED** [review]

File: modules/ROOT/pages/references/index.adoc [hunk 1]
```diff
@@ -2,8 +2,9 @@
 Mona Bärenfänger <mona@lightcurve.io>
 :description: An overview of the references available for the Lisk Elements packages and Lisk Commander.
 :toc:
-:page-previous: /lisk-sdk/v3/tutorials.html
+:page-previous: /lisk-sdk/v3/tutorials/index.html
 :page-previous-title: Tutorials
+:v_sdk: v3
```
Comment (Tschakki) [inline modules/ROOT/pages/references/index.adoc]: Why is this added?

File: modules/ROOT/pages/references/lisk-elements/index.adoc [hunk 2]
```diff
@@ -6,6 +6,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-previous: /lisk-sdk/v3/reference/lisk-commander/index.html
 :page-previous-title: Lisk Commander
 :title-logo-image: image:test-logo.png[]
+:v_sdk: v3
```
Comment (Tschakki) [inline modules/ROOT/pages/references/lisk-elements/index.adoc]: Why is this added?

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/references/index.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/references/index.adoc]: Now removed 👍

File: modules/ROOT/pages/references/lisk-elements/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/references/lisk-elements/index.adoc]: Also removed 👍

**Review by ishantiw — APPROVED** [review]

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**